src/libs/Navigation/AppNavigator/Navigators/Overlay.tsx

```diff
@@ -9,15 +9,18 @@ import CONST from '@src/CONST';
 type OverlayProps = {
     /* Callback to close the modal */
     onPress: () => void;
+
+    /* Returns whether a modal is displayed on the left side of the screen. By default, the modal is displayed on the right */
+    isModalOnTheLeft?: boolean;
 };
 
-function Overlay({onPress}: OverlayProps) {
+function Overlay({onPress, isModalOnTheLeft = false}: OverlayProps) {
```

Comment:
I think no need for a default value. `isModalOnTheLeft` will be undefined if not passed

Reply:
I think its cleaner to defined the default value than not.

Reply:
Value of isModalOnTheLeft should always be passed to the Overlay component. I've added the appropriate comment in the `OverlayProps` type with the information that we display the modal on the right side of the screen by default